segmentation/utils/spatial_analysis_utils_test.py

```diff
@@ -0,0 +1,182 @@
+import numpy as np
+import pandas as pd
+import random
+from segmentation.utils import spatial_analysis_utils
+from segmentation.utils import spatial_analysis
+import importlib
+importlib.reload(spatial_analysis)
+importlib.reload(spatial_analysis_utils)
+
+
+def test_calc_dist_matrix():
+    test_mat = np.zeros((1, 512, 512), dtype="int")
```

Comment:
Let's make a different easy pythagorean triple in a different slice of this array, just to make sure its preserving the information correctly, now that we're passing in multiple different labeled arrays